<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnifiedViewer</class>
 <widget class="QMainWindow" name="UnifiedViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YOLO + ResNet Unified Viewer with CPU &amp; NPU Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="videoLayout">
      <item>
       <widget class="QLabel" name="yolo_label">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resnet_label">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="yolo_info_label"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="cpu_info_label"/>
      </item>
      <item>
       <widget class="QLabel" name="npu_info_label"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
